MsGraphicsPkg/Library/SimpleUIToolKit/SimpleUIToolKit.inf

```diff
@@ -1,61 +1,60 @@
-## @file
-#  ManufacturingModeLib.inf
-#
-#  Description:    Library for determining if Manufacturing mode should be
-#                  enabled on this boot.  This library is what everyone should
-#                  use to determine which mode we are in.  It can be called from
-#                  PEI or DXE.
-#
-# Copyright (C) Microsoft Corporation. All rights reserved.
-# SPDX-License-Identifier: BSD-2-Clause-Patent
-##
-
-[Defines]
-  INF_VERSION                    = 0x00010005
-  BASE_NAME                      = UIToolKitLib
-  FILE_GUID                      = 4042708A-0F2D-4823-AC60-0D77B311188A
-  MODULE_TYPE                    = UEFI_DRIVER
-  VERSION_STRING                 = 1.0
-  LIBRARY_CLASS                  = UIToolKitLib|DXE_DRIVER DXE_RUNTIME_DRIVER DXE_SAL_DRIVER DXE_SMM_DRIVER UEFI_APPLICATION UEFI_DRIVER
-
-#
-#  VALID_ARCHITECTURES           = IA32 X64
-#
-
-[Sources]
-  Bitmap.c
-  Button.c
-  Canvas.c
-  EditBox.c
-  Grid.c
-  Label.c
-  ListBox.c
-  SimpleUIToolKit.c
-  ToggleSwitch.c
-  Utilities.c
-
-[Packages]
-  MdePkg/MdePkg.dec
-  MdeModulePkg/MdeModulePkg.dec
-  MsGraphicsPkg/MsGraphicsPkg.dec
-  MsCorePkg/MsCorePkg.dec
-
-[LibraryClasses]
-  DebugLib
-  BaseMemoryLib
-  MathLib
-  MsUiThemeLib
-  PcdLib
-  FltUsedLib
-
-[Protocols]
-  gMsOSKProtocolGuid
-
-[Guids]
-
-[Pcd]
-  gMsGraphicsPkgTokenSpaceGuid.PcdMsGopOverrideProtocolGuid
-
-[BuildOptions]
-  *_VS2013x86_*_CC_FLAGS       = /Oi /Og /fp:fast /arch:SSE
-#  *_VS2015x86_*_CC_FLAGS       = /Od /FAcs
+## @file
+#  SimpleUIToolKit.inf
+#
+#  Description:    Simple UI Toolkit (SUIT) library comprised of various UI elements that can be
+#                  used to draw user interfaces in the DXE environment.
+#
+# Copyright (C) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+[Defines]
+  INF_VERSION                    = 0x00010005
+  BASE_NAME                      = UIToolKitLib
+  FILE_GUID                      = 4042708A-0F2D-4823-AC60-0D77B311188A
+  MODULE_TYPE                    = UEFI_DRIVER
+  VERSION_STRING                 = 1.0
+  LIBRARY_CLASS                  = UIToolKitLib|DXE_DRIVER DXE_RUNTIME_DRIVER DXE_SAL_DRIVER DXE_SMM_DRIVER UEFI_APPLICATION UEFI_DRIVER
+
+#
+#  VALID_ARCHITECTURES           = IA32 X64
+#
+
+[Sources]
+  Bitmap.c
+  Button.c
+  Canvas.c
+  EditBox.c
+  Grid.c
+  Label.c
+  ListBox.c
+  ProgressBar.c
+  SimpleUIToolKit.c
+  ToggleSwitch.c
+  Utilities.c
+
+[Packages]
+  MdePkg/MdePkg.dec
+  MdeModulePkg/MdeModulePkg.dec
+  MsGraphicsPkg/MsGraphicsPkg.dec
+  MsCorePkg/MsCorePkg.dec
+
+[LibraryClasses]
+  DebugLib
+  BaseMemoryLib
+  MathLib
+  MsUiThemeLib
+  PcdLib
+  FltUsedLib
+
+[Protocols]
+  gMsOSKProtocolGuid
+
+[Guids]
+
+[Pcd]
+  gMsGraphicsPkgTokenSpaceGuid.PcdMsGopOverrideProtocolGuid
+
+[BuildOptions]
+  *_VS2013x86_*_CC_FLAGS       = /Oi /Og /fp:fast /arch:SSE
+#  *_VS2015x86_*_CC_FLAGS       = /Od /FAcs
```

Comment:
these old compiler flags should get cleaned up while you are here

Reply:
Thanks.  Removed.

Reply:
Looking into how to generically update the CC flags for all versions as these should be version-generic...

Reply:
does this work?
``` ini
MSFT:*_*_*_CC_FLAGS = <what ever you want here>
```

Reply:
The flags are no longer needed (or in some cases, even valid).  Removed.